site/irc.txt

```diff
@@ -0,0 +1,56 @@
+KISS IRC
+________________________________________________________________________________
+
+IRC was originally in the #kisslinux channel on Freenode, but was moved
+to Libera following the hostile behaviour on the part of Andrew Lee (see
+#/blog/20210512a).
+
+You can connect via Libera's webchat at https://web.libera.chat/#kisslinux.
+
+
+[1.0] Index
+________________________________________________________________________________
+
+- Public Logs                                                              [2.0]
+- IRC Etiquette                                                            [3.0]
+
+
+[2.0] Public Logs
+________________________________________________________________________________
+
+The IRC channel is publicly logged by a few of the channel denizens. Here are a
+list of public logs:
+
+- https://libera.irclog.whitequark.org/kisslinux/2021-05-29
+- (@phoebos) https://gemini.ctrl-c.club/~phoebos/logs/
+- (@cem) https://irclogs.carbslinux.org/kisslinux/
+- (@acheam) https://irclogs.armaanb.net/kisslinux/
+
+Unfortunately, the above sources don't contain any logs from before
+#kisslinux switched to Libera. For an archive of previous logs from
+Freenode, see https://freenode.logbot.info/kisslinux/.
+
+
+[3.0] IRC Etiquette
+________________________________________________________________________________
+
+The rules are simple: be excellent to each other, and use a pastebin
+when you want to share a long message.
+
+For those who need it spelled out:
+
+- Don't spam logs or error messages if they're longer than about 4 lines.
+  Use a pastebin (preferably https://0x0.st, as it doesn't require JS to
+  use).
+- When disagreeing, do so constructively [0][1].
+- Please refrain from discussing contentious topics such as US politics,
+  religion, or indentation styles in #kisslinux.
+- Racism, white supremacy sympathising, and the like will not be tolerated
+  under any circumstances.
```

Comment:
This is covered by \don't be shitty\ & contradicts our guidestones:
```
* There shall never be rules centred around speech or the way in which one must
  carry themselves to communicate. Do unto others as you would have them do
  unto you.
```

I understand that this is a very contentious topic. I understand problems relating to white supremacy, racism, dog whistles, subversion, micro aggressions, passive racism, ignorance, etc.
But we don't need to establish rules relating to this. 

Instead, we need to cultivate a community that respects these boundaries without being told that they exist. 

Building out infrastructure (rules, governance, bla bla bla) is plagued by bureaucratic redtape. The point of (our) rules (guidestones, in our case) is their flexibility and self-justification - the point of (generally) rules (laws, in that case) is their inflexibility, their scope, and their ability to make actions legally defensible. Rules exist so that when punishment is given, it is perceived as fair because it is governed by agreed to and well-established rules. We don't need this, because we are not a punitive system. I can already hear the complaints of "bla free speech bla" when we say 'non racists allowed'. The fact that racists should be afraid is, I think, self-evident (in at least my own social circles), and we are under no obligation to be kind to them -- our rules, after all, say nothing about us having to tolerate bad people. 

In short, we do not need rules telling them to go away. We can just be mean to them. 

I'll make it clear here and now (as I always try to) that I am unsympathetic to the perspectives of most people whom I view as disingenuous, morally bankrupt, intellectually dishonest, and vile. I have no problem telling racists to go away and that they are not welcome. I don't need a hammer to make their lives miserable in the circles I frequent. We need not use the cudgel of the law in contexts where the law does not apply. We have far more flexibility when we do not bind ourselves to rules.

Reply:
I think your judgement is fair on the second point. However, the statement

```
- Please refrain from discussing contentious topics such as US politics,  religion, or indentation styles in #kisslinux.
```

..isn't a command or a rule, but "hey, can we please keep the noise-to-signal ratio a bit lower?" Though I guess it could be rephrased to make that clear...

Reply:
I agree that aiming to keep noise-to-signal ratios low is good; specifying that the channel should be for tangentially KISS-related things is more valuable. By that I mean, we should specify that Linux-y topics are good, and that is what this channel is for. More KISS than anything else, but (at least personally speaking) I am fine with more broad sorts of topics in the tech space.

Reply:
Does the current rephrasing look good?